examples/recommendations/get_recommendation_impact_metrics.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example is to get impact metrics for a custom budget. 
+
+Use this example to get impact metrics for a budget amount during campaign creation workflows.
+
+This example uses the following: 
+1) Performance Max for the campaign type
+2) United States for the geo targeting 
+3) Maximize Conversions Value for the bidding strategy
+
+To get budget recommendations, run generate_budget_recommendations.py.
+"""
+
+
+import argparse
+import sys
+
+from google.ads.googleads.client import GoogleAdsClient
+from google.ads.googleads.errors import GoogleAdsException
+
+
+def main(client, customer_id, budget_amount_user):
+    """The main method that creates all necessary entities for the example.
+
+    Args:
+        client: an initialized GoogleAdsClient instance.
+        customer_id: a client customer ID.
+        budget_amount_user: a budget amount (not in micros) advertiser wants to use.
```

Comment:
Can you update the description to "a user-provided budget amount (not in micros), to retrieve impact metrics for."

Reply:
Will do